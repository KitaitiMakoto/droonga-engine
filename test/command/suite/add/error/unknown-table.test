#@include fixture/user-table.jsons
{
  "type": "add",
  "dataset": "Default",
  "body": {
    "table": "Unknown",
    "key": "key",
    "values": {
      "name": "Alice",
      "age": 20,
      "email": [
        "alice@example.com",
        "alice@example.jp"
      ]
    }
  }
}
