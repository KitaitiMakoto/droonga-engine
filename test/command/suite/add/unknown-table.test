#@include fixture/user-table.jsons
{
  "type": "add",
  "dataset": "Droonga",
  "body": {
    "table": "Unknown",
    "key": "key",
    "values": {
      "name": "Alice",
      "age": 20,
      "email": [
        "alice@example.com",
        "alice@example.jp"
      ]
    }
  }
}
