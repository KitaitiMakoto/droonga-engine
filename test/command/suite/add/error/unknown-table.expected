[
  "droonga.message",
  0,
  {
    "inReplyTo": "request-id",
    "statusCode": 404,
    "type": "add.result",
    "body": {
      "name": "UnknownTable",
      "message": "The table \"Unknown\" does not exist in the dataset."
    }
  }
]
